descriptors/ModuleDescriptor-template.json

```diff
@@ -291,6 +291,36 @@
           "pathPattern": "/acquisitions-units/units/{id}",
           "permissionsRequired": ["acquisitions-units.units.item.delete"],
           "modulePermissions": ["acquisitions-units-storage.units.item.delete"]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/acquisitions-units/memberships",
+          "permissionsRequired": ["acquisitions-units.memberships.collection.get"],
+          "modulePermissions": ["acquisitions-units-storage.memberships.collection.get"]
```

Comment:
Naming pattern differs from other permission names. You should use something like `orders.acquisitions-unit-memberships.XYZ`
`modulePermissions` naming also might be changed to be consistant with mod-orders-storage module. Please, take into account following comment
https://github.com/folio-org/mod-orders-storage/pull/130#discussion_r298043721

Reply:
@grabsefx ,
according discussion https://github.com/folio-org/mod-orders-storage/pull/130#discussion_r298136809 let it be as is for now. Are you OK with this?

Reply:
Sure, let it be as is.